frontend/test/metabase/scenarios/auth/search.cy.spec.js

```diff
@@ -0,0 +1,46 @@
+import {
+  restore,
+  signInAsAdmin,
+  signInAsNormalUser,
+  signIn,
+} from "__support__/cypress";
+
+describe("scenarios > auth > search", () => {
+  before(restore);
+
+  describe("universal search", () => {
+    it("should work for admin", () => {
+      signInAsAdmin();
+      cy.visit("/");
+      cy.findByPlaceholderText("Search…").type("product{enter}");
+      cy.findByText("PRODUCTS");
+    });
+
+    it.skip("should work for user with permissions (Issue #12332)", () => {
+      signInAsNormalUser();
+      cy.visit("/");
+      cy.findByPlaceholderText("Search…").type("product{enter}");
+      cy.findByText("PRODUCTS");
+    });
+
+    it("should not work for user without permissions", () => {
+      // Take away table permissions
+      signInAsAdmin();
+      cy.visit("/admin/permissions/collections");
```

Comment:
I thought that `signIn("nodata");` automatically would signin without any permissions, so why is all the revoking needed? And shouldn't it be visiting `/admin/permissions/databases`, since it's related to tables and not collections?

Reply:
I suppose it's not. I originally wrote the test to check whether a couple of normal searches worked correctly as well and must've left this in.
Thank you for catching this!